docs/Installation.md

```diff
@@ -699,6 +698,32 @@ database to use it with the new version of Zonemaster-Backend. Please see the
 
 For SQLite database upgrading is not needed as of now.
 
+## 8. Optional features
+
+### 8.1 Metrics

```

Comment:
There should be an entry in the table of contents for this. The update of this file will probably be in conflict with the update proposed by #855.

Reply:
> There should be an entry in the table of contents for this.

Done

> The update of this file will probably be in conflict with the update proposed by #855.

Yes I know, I will rebase when it's merged